creatures-hunting-service/src/test/java/cz/fi/muni/pa165/service/facade/UserFacadeUnitTest.java

```diff
@@ -0,0 +1,182 @@
+package cz.fi.muni.pa165.service.facade;
+
+import cz.fi.muni.pa165.dto.UserAuthenticateDTO;
+import cz.fi.muni.pa165.dto.UserDTO;
+import cz.fi.muni.pa165.entity.User;
+import cz.fi.muni.pa165.enums.UserRole;
+import cz.fi.muni.pa165.facade.UserFacade;
+import cz.fi.muni.pa165.service.BeanMappingService;
+import cz.fi.muni.pa165.service.UserService;
+import cz.fi.muni.pa165.service.config.ServiceConfiguration;
+import org.mockito.InjectMocks;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Balcirak Peter <peter.balcirak@gmail.com>
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class UserFacadeUnitTest extends AbstractTestNGSpringContextTests {
+
+	private UserService userService = mock(UserService.class);
+
+	@Inject
+	private BeanMappingService beanMappingService;
```

Comment:
I think that correct unit test should use only mocks. Even the Bean mapping service. But I think that you dont need to change it.